ui/admin/tests/e2e/global-setup.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-undef */
```

Comment:
**For visibility:** We shouldn't be ignoring the linter here and that's NOTHING to do with this PR. This is a tech debt item we need to solve. For e2e that run using `playright` (so node js) we have to write CommonJS but for the rest of our code that runs in the browser we have to write modules.

So we need to discuss that as a team and if we keep both types we should change linter configuration to allow Common JS just for tests.

As I said earlier, NOT related to this PR and NOT to be fixed in this PR.